crypto/ui/ui_lib.c

```diff
@@ -427,9 +427,13 @@ static int print_error(const char *str, size_t len, UI *ui)
 int UI_process(UI *ui)
 {
     int i, ok = 0;
+    const char *state = NULL;
 
-    if (ui->meth->ui_open_session && !ui->meth->ui_open_session(ui))
-        return -1;
+    if (ui->meth->ui_open_session && !ui->meth->ui_open_session(ui)) {
```

Comment:
Only there?  This is old stuff and that kind of change is needed all over the place.  I'd say that's for a separate cleanup PR.